homeassistant/components/synology_dsm/config_flow.py

```diff
@@ -112,8 +112,9 @@ def __init__(self) -> None:
         self.reauth_conf: dict[str, Any] = {}
         self.reauth_reason: str | None = None
 
-    async def _show_setup_form(
+    async def _show_form(
```

Comment:
The form helper doesn't need to be a coroutine function since we don't await inside.